features/admin.connections.v1/components/edit/forms/authenticators/saml-authenticator-form.tsx

```diff
@@ -959,6 +961,25 @@ export const SamlAuthenticatorSettingsForm: FunctionComponent<SamlSettingsFormPr
                             readOnly={ readOnly }
                         />
                     </SectionRow>
+                    <SectionRow>
+                        <Field.Input
+                            name="authnReqProviderName"
+                            value={ formValues?.AuthnReqProviderName }
+                            inputType="default"
+                            placeholder={ t(`${ I18N_TARGET_KEY }.authnReqProviderName.placeholder`) }
+                            ariaLabel={ t(`${ I18N_TARGET_KEY }.authnReqProviderName.ariaLabel`) }
+                            data-testid={ `${ testId }-authnReqProviderName-field` }
+                            label={ (
+                                <FormInputLabel htmlFor="authnReqProviderName">
+                                    { t(`${ I18N_TARGET_KEY }.authnReqProviderName.label`) }
+                                </FormInputLabel>
+                            ) }
+                            maxLength={ 100 }
```

Comment:
Is there a specific reason to limit length to 100? can there be instances where this value can exceed 100 characters?

Reply:
I got the following error when removing the maxLength attribute, so I set a max value. Since this is expected to be the human readable name of the provider, for example, "Travelocity", "WSO2 Identity Server", which would not be very long, I added 100 as the limit.

<img width="617" alt="Screenshot 2024-12-19 at 13 25 28" src="https://github.com/user-attachments/assets/e151936b-e89e-4280-bed0-969d674fff6b" />

Reply:
As discussed offline, updated to 240 with [6168cf4](https://github.com/wso2/identity-apps/pull/7225/commits/6168cf41b51f917e7fea3e3e9edaa0a039931571).